config/jobs/ppc64le-cloud/all-in-one/allinone-postsubmit.yaml

```diff
@@ -0,0 +1,91 @@
+postsubmits:
+  ppc64le-cloud/test-infra:
+    - name: postsubmit-allinone-job
+      decorate: true
+      run_if_changed: 'images/all-in-one'
+      branches:
+        - master
+      spec:
+        serviceAccountName: build-img
+        containers:
+          - image: debian:buster
```

Comment:
why have you used this image vs a different one for all the remaining ones?

Reply:
> why have you used this image vs a different one for all the remaining ones?

In the other jobs, we were using all-in-one image but here we are creating a job for all-in-one image itself

Reply:
In the other jobs, we were using all-in-one image but here we are creating a job for all-in-one image itself.